packages/nextjs/package.json

```diff
@@ -51,6 +54,7 @@
     "@types/react-copy-to-clipboard": "^5.0.4",
     "@typescript-eslint/eslint-plugin": "^5.39.0",
     "autoprefixer": "^10.4.12",
+    "bun": "^1.0.30",
```

Comment:
added bun to be able to easily run the `seed.ts` file

Reply:
The build fails because bun is not installed. I think you need to add a vercel.json file that explicitly installs bun like here: https://maximevermeeren.medium.com/how-to-use-bun-on-vercel-7512383153d7

Since we are only using bun to run the seed.ts file you shouldn't need to make it run the build command with bun. Just install it.